frontend/__tests__/agency-form.test.js

```diff
@@ -28,9 +28,9 @@ describe("AgencyProfileForm.handleInputChange", () => {
     expect(component.state.agencyNumber).toBe("12345");
 
     // another string value
-    expect(component.state.agencyName).toBe("");
-    component.handleInputChange("agencyName", "Test Name");
-    expect(component.state.agencyName).toBe("Test Name");
+    expect(component.state.name).toBe("");
+    component.handleInputChange("name", "Test Name");
+    expect(component.state.name).toBe("Test Name");
```

Comment:
Why was agencyName changed to name here?

Reply:
See below! :-)